src/db/util/raw-model.ts

```diff
@@ -181,6 +195,15 @@ export const defineRawModel =
       return res.map(validateAndFilter);
     };
 
+    const count: CountFn<F> = async ({ where, trx } = {}) => {
+      const builder = trx ? tbl().transacting(trx) : tbl();
+      const query = builder.where(prepareCondition(where ?? {})).count('*');
+
+      const res = await query;
+
+      return res.map((val) => val as unknown);
+    };
```

Comment:
The existence of `count` method isn't really justified. It doesn't provide any value. You can do the exact same thing by using `find` and then checking `length` property of results.

The mapping `res.map((val) => val as unknown)`, which you do at the end of this method isn't doing anything actually. You are just looping through an array and returning its elements.

Reply:
After different changes through development I agree with you that this is not longer needed.
The intention of doing this was to reduce execution times when performing large search queries while pagination.
Doing this the execution time was reduced notably. 
Nonetheless This is no longer needed - I'll remove it.